### PR TITLE
indexing: update document and ebooks mapping

### PR DESCRIPTION
**Pull request by Garfield-fr**
* Adds specific languages analyzers to the template.
* Adds analyzer on documents: title, titlesProper, is_part_of, authors.name, publishers.name, series.name, notes.name, subjects
* Adds analyzer on ebooks: title, titlesProper, is_part_of, authors.name, publishers.name, series.name, abstracts
* Improves facet authors (languages)

How to test:

* `script/bootstrap`
* `script/setup`
* http://localhost:5000
* Find some terms and check if the result is correct

Co-Authored-by: Bertrand Zuchuat <bertrand.zuchuat@rero.ch>

### CODE REVIEW

File: rero_ils/es_templates/v6/record.json [hunk 1]
```diff
@@ -1,33 +1,125 @@
...
+          "tokenizer": "char_group_tokenizer",
+          "filter": [
+            "lowercase",
+            "english_stop",
+            "english_snowball",
+            "asciifolding",
+            "english_stemmer"
+          ]
+        },
+        "french_analyzer": {
```
Comment (jma) [inline rero_ils/es_templates/v6/record.json]: Please use the standard language analyzer.

Comment (Garfield-fr) [reply rero_ils/es_templates/v6/record.json]: done

File: rero_ils/modules/documents/mappings/v6/documents/document-v0.0.1.json [hunk 2]
```diff
@@ -121,7 +183,26 @@
           "type": "object",
           "properties": {
             "name": {
-              "type": "text"
+              "type": "text",
```
Comment (jma) [inline rero_ils/modules/documents/mappings/v6/documents/document-v0.0.1.json]: publisher is a name thus we do not need languages analyzers.

File: rero_ils/modules/documents/mappings/v6/documents/ebook-v0.0.1.json [hunk 3]
```diff
@@ -59,14 +136,42 @@
...
+          "type": "keyword"
+        },
+        "facet_authors_it": {
           "type": "keyword"
         },
         "publishers": {
           "type": "object",
           "properties": {
             "name": {
-              "type": "text"
```
Comment (jma) [inline rero_ils/modules/documents/mappings/v6/documents/ebook-v0.0.1.json]: same as documents

File: rero_ils/modules/documents/mappings/v6/documents/document-v0.0.1.json (hunk 2 above)
Comment (Garfield-fr) [reply rero_ils/modules/documents/mappings/v6/documents/document-v0.0.1.json]: done

File: rero_ils/modules/documents/mappings/v6/documents/ebook-v0.0.1.json (hunk 3 above)
Comment (Garfield-fr) [reply rero_ils/modules/documents/mappings/v6/documents/ebook-v0.0.1.json]: done

**Pull request merged**